tools/vms-generator/vms-generator.go

```diff
@@ -96,6 +96,7 @@ func getBaseVm(name string) *v1.VirtualMachine {
 		},
 		ObjectMeta: metav1.ObjectMeta{
 			Name: name,
+			Labels: map[string]string{"special": "key"},
```

Comment:
this is needed for service exposure to work. following: https://github.com/kubevirt/user-guide/blob/master/workloads/virtual-machines/expose-service.md